### ISSUE TITLE
Workspace Toolbar Inconsistencies

### ISSUE THREAD

**Issue by ReinerBforartists** [Bforartists/Bforartists#4161]
With the new toolbar system a few inconsistencies has arrived. The main layout should imo not contain the animation toolbar content visible by default. At least not the keying set edit box. This box resides visibly down right already. Also the add and remove keyframe button. 

In the Animation layout we have a double problem. Same set of tools visible. We might even remove the second toolbar here in case we keep the animation stuff in the main toolbar on. Visible Doubles in the default UI is not good. It just adds visual noise.

There is also a gap here and there below the separated toolbars.

In short, go through all the layouts, and check for inconsistencies and double displayed content.

![inconsistencies](https://github.com/Bforartists/Bforartists/assets/13919702/a8aaae5b-0ff6-4631-8964-901dd57d8dad)

**Comment by ReinerBforartists** [comment]
I would say turn off the animation stuff in the main toolbar (it is specially made for animation, so no need to show it all the time in the general layout), keep the second toolbar for animation needs in the animation layout, and turn off the keying set prop. It is already down right as told.

There is more inconsistency from what i have seen. In the UV Layout for example. That's why i said go through all workspaces, and look first ...

![uvlayout](https://github.com/Bforartists/Bforartists/assets/13919702/538d45ea-d10b-4df0-91d1-bda90cd16131)

**Comment by ReinerBforartists** [comment]
> For the gaps on the extra toolbar editors, I saved in W11 on a 1080p 16:9. If you have a taller monitor you will have to do that save.. or I can test a version without the toolbar to "compress" the second toolbars?

You could separate the part with the gap. Then i'll have a look. It's just, you need to resave the layouts for the toolbar issue. So i thought i mention it here already.

I have a 2560 screen now, not a 1920 anymore :)

**Comment by Draise14** [comment]
Ah ok, I'll let you do it on the larger monitor. I don't have a budget to buy a higher dpi monitor..

**Comment by ReinerBforartists** [comment]
I had to, the light went off ^^

**Comment by Blenux** [comment]
Wish there was a way to make the Editor Toolbar a fixed size :thinking: 

Also, how about a toggle to display in certain Workspaces, perhaps?

https://github.com/Bforartists/Bforartists/assets/25260650/e5d51e4d-1ca2-46ca-b1d3-f01ca54f9cd7

**Comment by ReinerBforartists** [comment]
Don't make it too complicated. The most important design rule is KISS. Keep it simple, stupid  ;)

**Comment by Blenux** [comment]
Just a thought that's one major difference between the Top Toolbar and the Editor Toolbar, the editor toolbar come have multiple versions of itself when different options enabled or disabled, even in different workspaces, where the Top Toolbar can't.

Hope that makes sense :smile:

**Comment by ReinerBforartists** [comment]
Indeed. And that's why we have to keep it slim.

**Comment by Draise14** [comment]
![bforartists_hsfYqFUTIU](https://github.com/Bforartists/Bforartists/assets/35357409/344d9190-2cee-4c98-bbd6-9f4adb0376fa)

I guess defaults can have these two off (as these layouts have the second toolbar editor for them)

**Comment by ReinerBforartists** [comment]
Nobody is assigned, i want to close the gap anyways, so let's tackle this one.

**Comment by ReinerBforartists** [comment]
Hmm. I am baffled. Is changing the main toolbar types not longer bound to the startup.blend ?

**Comment by ReinerBforartists** [comment]
Closed the gaps

**Comment by ReinerBforartists** [comment]
Ah i see, the toptoolbar uses python. KK.

Fixed.
